client/components/gsuite/gsuite-new-user-list/index.tsx

```diff
@@ -57,6 +61,13 @@ const GSuiteNewUserList: FunctionComponent< Props > = ( {
 	};
 
 	const onUserAdd = () => {
+		if ( context === 'inbox-management' ) {
+			recordInboxUpsellEvent( {
```

Comment:
BTW why are we recording an inbox upsell event here? This is just a change event that happens any time the users loads a new mailbox form. The user can still remove this mailbox without checking out.

Reply:
In the Asan task this event should be recorded. Maybe I understood bad, but as per this text is what I understood:
> User clicks on the button to create a new mailbox